openfisca_us/variables/gov/states/ma/tax/income/credits/ma_limited_income_tax_credit.py

```diff
@@ -0,0 +1,37 @@
+from openfisca_us.model_api import *
+
+
+class ma_limited_income_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MA limited income tax credit"
+    unit = USD
+    definition_period = YEAR
+    is_eligible = in_state("MA")
+    reference = "https://www.mass.gov/doc/2021-schedule-nts-l-nrpy-no-tax-status-and-limited-income-credit/download"
+
+    def formula(tax_unit, period, parameters):
+        # Line numbers refer to MA's 2021 Schedule NTS-L-NR/PY.
+        # Line 10.
+        agi = tax_unit("ma_agi", period)
+        exempt = tax_unit("is_state_income_tax_exempt", period)
+        # Lines 10-12: Compute eligibility based on income limit.
+        state = tax_unit.household("state_code_str", period)
+        filing_status = tax_unit("filing_status", period)
+        tax = parameters(period).states.ma.tax.income
+        p = tax.credits.limited_income
+        filer_limit = p.income_limit.filer[state][filing_status]
```

Comment:
Could you remove `state` as a breakdown from this parameter, since it's now in the `ma` folder?